src/common/locations.ts

```diff
@@ -1,17 +1,13 @@
 /** Helpers for dealing with the State / Counties dataset. */
-import has from 'lodash/has';
-import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
-import countyAdjacencyMsa from './data/county_adjacency_msa.json';
 
-interface AdjacencyData {
-  [fips: string]: {
-    adjacent_counties: string[];
-    msa_code?: string;
-  };
+export interface Location {
+  state_fips_code: string;
+  full_fips_code?: string;
+  population: number;
+  state_code: string;
+  state: string;
 }
 
-const ADJACENT_COUNTIES: AdjacencyData = countyAdjacencyMsa.counties;
-
 /** Aggregations supported by the Explore chart. */
 export const AGGREGATED_LOCATIONS: Location[] = [
```

Comment:
FWIW- We should be able to delete this in favor of https://github.com/covid-projections/covid-projections/blob/398c97144382962d4dc97827d65dbc118b46cb8b/src/common/regions/preprocessed_regions_data.ts#L33 and this whole file could go away!  (but feel free to punt. you've done enough great cleanup in this PR. 😁 )

Reply:
good catch, I'll look into it.

Reply:
yep, this was pretty straightforward.  it turns out, since customareas are 'state' regions, we can just use their fullname directly and don't need to even bring in customAreasByFips in the place it's used.